client/src/common/components/Dialog.jsx

```diff
@@ -67,8 +66,8 @@ const Dialog = ({
 );
 
 Dialog.defaultProps = {
-  cancelLabel: 'common.button.cancel',
-  confirmLabel: 'common.button.confirm',
+  cancelLabel: 'cancel',
```

Comment:
if someone does not pass those labels he will get English ones - they are hardcoded now...